contracts/token/CRC1155/extensions/CRC1155Enumerable.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity ^0.8.0;
 import "./ICRC1155Enumerable.sol";
 import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
 abstract contract CRC1155Enumerable is ERC1155, ICRC1155Enumerable {
```

Comment:
Does this contract need to inherit IERC165 interface?

Reply:
Yes, the `IERC165` interface is required by compiler when overriding the `supportsInterface` method.